src/extension/background-script/connectors/alby.ts

```diff
@@ -254,11 +254,18 @@ export default class Alby implements Connector {
         token: this.config.oAuthToken, // initialize with existing token
       });
 
+      authClient.on("tokenRefreshed", async (token: Token) => {
+        await this._updateOAuthToken(token);
+      });
+
+      authClient.on("tokenRefreshFailed", async (error: Error) => {
+        console.error("Failed to Refresh token", error);
```

Comment:
It would be nice to comment here what happens when the refresh fails in the different scenarios. Can we make sure they are always handled correctly (e.g. the auth modal is re-launched)

Reply:
Maybe add a comment:

```
// Currently the JS SDK guarantees request of a new refresh token is done synchronously.
// The only way a refresh should fail is if the refresh token has expired, which is handled when the connector is initialized.
// If a token refresh fails after init then the connector will be unusable, but we will still log errors here so that this can be debugged if it does ever happen.
```

Reply:
resolved